DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -104,6 +104,18 @@ function PreFirstEntryMapFixes()
                     break;
                 }
             }
+
```

Comment:
In theory I would like to see this wrapped in a check to ensure the map is only mirrored in the X direction (ie. check that the multiplier has a zero Y and Z component), but it's not critical